### DISCUSSION TITLE
any reason why the apps are licensed as gpl3 and not agpl3?

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [FossifyOrg/General-Discussion#130] [General]
?

**Comment by naveensingh** [comment]
@RustoMCSpit any reason why the apps should be licensed as AGPL? 

As far as I know, AGPL is more suitable for things like [websites](https://github.com/FossifyOrg/fossifyorg.github.io). Also, relicensing is changing the terms, and that comes with caveats.

**Reply by inson1** [reply]
its stronger license  (in term of free software), but its important for server things (If I get it correctly) (but shouldnt be weaker then GPL in any way)
